src/onnx/parse_layernorm.cpp

```diff
@@ -0,0 +1,47 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/op/layernorm.hpp>
+#include <migraphx/argument.hpp>
+#include <migraphx/instruction.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_layernorm : op_parser<parse_layernorm>
+{
+    std::vector<op_desc> operators() const { return {{"LayerNormalization"}}; }
+
+    instruction_ref parse(const op_desc& /*opd*/,
+                          const onnx_parser& parser,
+                          onnx_parser::node_info info,
+                          const std::vector<instruction_ref>& args) const
+    {
+        float epsilon = 1e-3f;
+        int64_t axis  = -1;
+        if(contains(info.attributes, "epsilon"))
+        {
+            epsilon = parser.parse_value(info.attributes.at("epsilon")).at<float>();
+        }
+        if(contains(info.attributes, "axis"))
+        {
+            epsilon = parser.parse_value(info.attributes.at("axis")).at<int64_t>();
+        }
+
+        auto layernorm = info.add_instruction(
+            make_op("layernorm", {{"epsilon", epsilon}, {"axis", axis}}), args.front());
```

Comment:
We shouldn't be adding a custom layernorm operator. We should just insert layernorm using our existing operators.

Reply:
That is something I had done previously, but for some reason the fusion of those ops into gpu::layernorm and gpu::triaddlayernorm wasn't happening inside ORT even though it was happening in the driver. I can try it again and see if I can figure out why.

Reply:
Yea we need to investigate why this fusion is not happening. #1212 did do some improvements for the matcher, I am not sure if that was related or not.